thinkhazard/security.py

```diff
@@ -0,0 +1,12 @@
+import os
+from passlib.apache import HtpasswdFile
+
+
+def groupfinder(username, password, request):
+    ht = HtpasswdFile()
+    ht.load_string(os.environ.get("HTPASSWORDS", ""))
+
+    if ht.check_password(username, password):
```

Comment:
I guess decrypting passwords takes place here?

Reply:
Yes, but passwords cannot be decrypted (it would be a severe security issue !).
Passwords are encrypted and compared to the already encrypted ones stored in htpassword file, or env var in our case.